docs/src/index.ejs

```diff
@@ -26,27 +26,24 @@
   </script>
 </head>
 <body>
-  <script src="//cdnjs.cloudflare.com/ajax/libs/anchor-js/3.2.1/anchor.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/anchor-js/3.2.1/anchor.min.js"></script>
```

Comment:
### Why did I normalize all scripts to `//`?
Scripts without the protocol (http / https) allow the browser to request the script using the current protocol.  In dev mode, http, this would allow scripts to be fetched over http which is much faster than https.  This is valid HTML per the specifications as you can see [here](https://stackoverflow.com/questions/550038/is-it-valid-to-replace-http-with-in-a-script-src-http).

### Which should we use?

In fact, it seems I needed to update my best practices. You can see more from Paul Irish [here](https://www.paulirish.com/2010/the-protocol-relative-url/), but the short answer is, https is encouraged for all sources, it is no longer a perf concern, and protocol-less urls are considered an anti-pattern.

Your changes here are correct.